android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -1759,130 +1765,7 @@ public void run() {
-                        moveSeekBarIndicator(seekbar, isRew);
-
-                        animateHideView(currentTextView, 200);
-                    }
-
-                    keyNotHandled = false;
-                    break;
-            }
-        } else if (event.getAction() == KeyEvent.ACTION_UP) {
-
-            if (keyNotHandled && previewSeekBarLayout.getPreviewView() instanceof SeekBar && ((SeekBar) previewSeekBarLayout.getPreviewView()).hasFocus()) {
-                switch (event.getKeyCode()) {
-                    case KeyEvent.KEYCODE_DPAD_LEFT:
-                    case KeyEvent.KEYCODE_MEDIA_REWIND: {
-                        if (player == null || !areControlsVisible) {
-                            break;
-                        }
-
-                        long position = player.getCurrentPosition() - 10000;
-                        if (position < 0) {
-                            position = 0;
-                        }
-                        controlDispatcher.dispatchSeekTo(player, player.getCurrentWindowIndex(), position);
-                        //updateProgressControl(position);
-                        seekIndicator.show(true, getSeekBarPositionString(player.getCurrentPosition(), player.getDuration()), 1000, seekIndicatorRunnable);
-                        moveSeekBarIndicator((SeekBar) previewSeekBarLayout.getPreviewView(), true);
-                        animateHideView(currentTextView, 200);
-                        break;
-                    }
-                    case KeyEvent.KEYCODE_MEDIA_FAST_FORWARD:
-                    case KeyEvent.KEYCODE_DPAD_RIGHT: {
-                        if (player == null || !areControlsVisible) {
-                            break;
-                        }
-
-                        long position = player.getCurrentPosition() + 10000;
-                        if (position > player.getDuration()) {
-                            position = player.getDuration();
-                        }
-                        controlDispatcher.dispatchSeekTo(player, player.getCurrentWindowIndex(), position);
-                        //updateProgressControl(position);
-                        seekIndicator.show(false, getSeekBarPositionString(player.getCurrentPosition(), player.getDuration()), 1000, seekIndicatorRunnable);
-                        moveSeekBarIndicator((SeekBar) previewSeekBarLayout.getPreviewView(), false);
-                        animateHideView(currentTextView, 200);
-                        break;
-                    }
-                }
-            }
-
-            if (!isPaused) {
-                hideOverlay();
-            }
-
-            keyPressTime = null;
-            keyNotHandled = false;
-        }
-
-        return super.dispatchKeyEvent(event);
+        return exoDorisPlayerView.dispatchKeyEvent(event) || super.dispatchKeyEvent(event);
```

Comment:
are we sure there are no regressions around using remote buttons during playback ?